.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+Please ensure that the following information is filled before filing
+--------------------------------------------------------------------
+
+Please note that the alot version should be either the latest released version,
+or git sha for a development version newer than the most recent release
+
+Please run alot again with the options `-d debug -o log`
+
+Software Versions
+-----------------
+
+Python version:
+alot version:
+
+Observed behavior
+-----------------
+
+Please include any tracebacks or error information in the log file that you
+generated, as well as a description of the behavior
```

Comment:
fullstop. And then maybe one more sentence: "Please don't post the whole log file blindly as it might contain sensitive information and might be quite big."